.buildkite/pipeline.yml

```diff
@@ -53,3 +53,16 @@ steps:
           workdir: /repo
           mount-ssh-agent: true
           command: ["--confirm", "release-deploy", "--from-label", "latest", "--environment-id", "stage", "--description", $BUILDKITE_BUILD_URL]
+  - wait
+  - label: send a test bag in stage
+    if: build.branch == "master"
+    plugins:
+      - docker#v3.5.0:
+          image: "amazon/aws-cli"
+          mount-ssh-agent: true
```

Comment:
I'm not certain you need this.

Reply:
Ah, possibly not. Do AWS credentials get exposed inside the container then?

Reply:
The containers can access the host credentials via the network endpoint available on EC2 instances.

Reply:
So basically it's transparent to anything that uses an AWS SDK.